### ISSUE TITLE
wc-edostavka woocommerce 3.0.7

### ISSUE THREAD

**Issue by mel139** [kalbac/wc-edostavka#41]
Максим, здравствуйте.
Установил плагин на тестовом сайте с woo 2.6.4, все отлично, плагин просто супер. Но на основном сайте стоит уже woo 3.0.7 и откатываться назад страшновато, т.к. обязательно что-нибудь перестанет работать. В СДЭК при запросе плагина перенаправляют на вас. Вопрос я смотрю актуальный для многих, аналогов этого плагина я нигде не нашел.  Планируется ли обновление плагина для работы с woo 3 (если да, то когда?) и будет ли дальнейшая поддержка плагина? 
Спасибо!

**Comment by kalbac** [comment]
Здравствуйте. Дальнейшая бесплатная поддержка плагина больше не будет осуществляться. На данный момент адаптирую плагин под 3+ версию вуккомерца только по запросу пользователей на платной основе.

**Comment by mel139** [comment]
Сколько будет стоить адаптация?

29 мая 2017 г., 14:27 пользователь Максим Мартиросов <
notifications@github.com> написал:

**Comment by mel139** [comment]
И как с вами связаться?

**Comment by kalbac** [comment]
maksim@martirosoff.ru

**Comment by miraserver** [comment]
а нельзя ли просто выпустить коммерческую версию с фиксированной
ценой...чтобы было всем понятна совместимость, требования, возможности. А
то как-то не по -программистки, честное слово:)) Один раз поработать,
пофиксить баги, протестировать на совместимость, обкатать функционал,
благо, не с нуля все и не на пустом месте. А потом просто ждать, пока
денежки капают и оказывать support, ну там на 6 мес включенный в
стоимость...
На codecanyon.net выложить и нормально...

2017-05-29 14:59 GMT+03:00 Максим Мартиросов <notifications@github.com>:

**Comment by kalbac** [comment]
Да, это идея есть в планах. Но я всё никак не найду свободного времени что бы сделать всё это поуму. Когда делаешь плагин под широкую аудиторию, нужно учитывать много нюансов. А когда делаешь под конкретного пользователя на конкретном сервере, то и отталкиваешься только от нужд клиента.

**Comment by vguns** [comment]
почему select2 не работает в новом вукомерце?
